tools/eslint-rules/rules-utils.js

```diff
@@ -12,6 +12,16 @@ module.exports.isRequired = function(node, modules) {
     modules.includes(node.arguments[0].value);
 };
 
+/**
+* Return true if common module is required
+* in AST Node under inspection
+*/
+var commonModuleRegExp = new RegExp(/^(\.\.\/)*common(\.js)?$/);
+module.exports.isCommonModule = function(node) {
+  return node.callee.name === 'require' &&
+    commonModuleRegExp.test(node.arguments[0].value);
```

Comment:
This will produce access errors due to accessing nodes without arguments. Please guard against them the same as I suggested in the other PR.